packages/client/python-psutil/python-psutil.spec

```diff
@@ -57,7 +53,8 @@ ifconfig, nice, ionice, iostat, iotop, uptime, pidof, tty, who, taskset, pmap.
 
 
 %prep
-%autosetup -p0 -n %{srcname}-release-%{version}
+%setup -q -n %{srcname}-release-%{version}
+%patch0 -p1
```

Comment:
Technically you could have left the %autosetup here, with -p1. But no hard preference either way from me.

Reply:
Sold!